packages/manager/apps/hub/src/index.html

```diff
@@ -2,6 +2,10 @@
 <html>
     <head>
         <title>ManagerHub App</title>
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1, shrink-to-fit=no"
```

Comment:
```suggestion
            content="width=device-width, initial-scale=1"
```
You can almost certainly remove it :)

Reply:
This is done :)